packages/utils/models.js

```diff
@@ -4,15 +4,24 @@ const ThisStore = {
   _storedModel: undefined,
 };
 
+/**
+ * @description Check if method is available via Mamba
+ * @returns {boolean}
+ */
 function VerifyMethodOnSystemWrapper(method) {
+  if (!__POS__) return null;
+
```

Comment:
Tive que mudar pra undefined, porque se o método do back retornar `false` `!value`, ele vai cair na lista estática, que é o que não queremos.

```suggestion
  if (!__POS__) return undefined;

```